deployment/deploy-core-infrastructure.ps1

```diff
@@ -20,14 +20,18 @@ Param(
 
     [Parameter(mandatory=$true)]
     [string]
-    $SubnetAddressPrefix
+    $SubnetAddressPrefix,
+
+    [Parameter(Mandatory = $false)]
+    [bool]
+    $SetupArc = $true
 )
 
 # Uncomment this if you are testing this script without deploy-az-demo-bootstrapper.ps1
 # Import-Module -Name .\modules\text-utils.psm1
 
 class Aks {
-    [PSCustomObject] Prepare ([string]$resourceGroupName, [string]$aksName, [PSCustomObject]$proxyConfig){
+    [PSCustomObject] Prepare ([string]$resourceGroupName, [string]$aksName, [PSCustomObject]$proxyConfig, [bool]$EnableArc){
```

Comment:
nit: casing of parameter could be same as others.

Reply:
Agree